src/pepr/operator/controllers/network/generate.ts

```diff
@@ -7,6 +7,56 @@ import { cloudMetadata } from "./generators/cloudMetadata";
 import { intraNamespace } from "./generators/intraNamespace";
 import { kubeAPI } from "./generators/kubeAPI";
 
+function isWildcardNamespace(namespace: string) {
+  return namespace === "" || namespace === "*";
+}
+
+function getPeers(policy: Allow): V1NetworkPolicyPeer[] {
+  let peers: V1NetworkPolicyPeer[] = [];
+
+  if (policy.remoteGenerated) {
+    switch (policy.remoteGenerated) {
+      case RemoteGenerated.KubeAPI:
+        peers = kubeAPI();
+        break;
+
+      case RemoteGenerated.CloudMetadata:
+        peers = cloudMetadata;
+        break;
+
+      case RemoteGenerated.IntraNamespace:
+        peers = [intraNamespace];
+        break;
+
+      case RemoteGenerated.Anywhere:
+        peers = [anywhere];
+        break;
+    }
+  } else if (policy.remoteNamespace !== undefined || policy.remoteSelector !== undefined) {
```

Comment:
Just noting this is technically a change in behavior - previously someone could chain together both remoteGenerated and remoteNamespace/Selector and end up with a policy with 2 peers I think?

I think this approach is more sane, require each list item to have one or the other? We may want to add to the [schema](https://github.com/defenseunicorns/uds-core/blob/main/src/pepr/operator/crd/sources/package/v1alpha1.ts#L39) though (oneOf for this).

Reply:
Missed [this thread](https://github.com/defenseunicorns/uds-core/pull/564#discussion_r1692181399) earlier - resolving that one to continue any discussion here.

Reply:
Forgot about the validation we are already doing [here](https://github.com/defenseunicorns/uds-core/blob/fix-inter-ns-netpol/src/pepr/operator/crd/validators/package-validator.ts#L61-L64) so this is not actually a shift in functionality and is already being caught in the validator.